index.css

```diff
@@ -0,0 +1,214 @@
+/* stylelint-disable max-empty-lines */
```

Comment:
- [ ] Please, consider removing all linter disables from your code. Linter should make its direct job - to help you find bugs for fix them on the beginning stage of your project 👍